custom_components/mqtt_discoverystream/__init__.py

```diff
@@ -5,11 +5,7 @@
 
 import voluptuous as vol
 
-from homeassistant.components.mqtt import (
-  valid_publish_topic,
-  async_subscribe,
-  async_publish,
-)
+from homeassistant.components import mqtt
```

Comment:
Moved the above imports to be part of the overall mqtt import to make it more obvious where the functions were coming from